packages/web-app-files/src/components/Collaborators/CollaboratorsEditOptions.vue

```diff
@@ -1,13 +1,18 @@
 <template>
   <div>
-    <translate tag="label" for="files-collaborators-role-button" class="oc-label">Role</translate>
+    <translate
+      :aria-label="$gettext('Role')"
+      tag="label"
+      for="files-collaborators-role-button"
+      class="oc-label"
+      >Role</translate
```

Comment:
Could you run `yarn lint` in the project please? The linter failed in CI, this command should show you what needs to be changed (and you can `yarn lint --fix` to fix it)